### PR TITLE
Fix unexpected duplicate task error for new Xcode build system

### PR DESCRIPTION
**Pull request by maks-g**
Fix compilation of the project when using Xcode 9 new build system and CocoaPods.
Details can be found here: https://github.com/CocoaPods/CocoaPods/issues/7079

### CODE REVIEW

**Comment by maks-g** [comment]
Hi!yes you are right

File: Source/Bundle+Convenience.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+extension Bundle {
+    class var resources: Bundle {
```
Comment (sberrevoets) [inline Source/Bundle+Convenience.swift]: Can you make this `static` for consistency with the rest of the project? Also, I'd like for this to be called `resourceBundle` as `resources` doesn't make it very clear that this a `Bundle` is being returned (as opposed to, say, an array of resources).

File: Source/Bundle+Convenience.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        let frameworkBundle = Bundle(for: AlertController.self)
+
+        if let bundleURL = frameworkBundle.url(forResource: "SDCAlertView", withExtension: "bundle"),
+            let bundle = Bundle(url: bundleURL) {
+            return bundle
+        }
```
Comment (sberrevoets) [inline Source/Bundle+Convenience.swift]: I think we can make this a little shorter:

```swift
let sourceBundle = Bundle(for: AlertController.self)
let bundleURL = sourceBundle.url(forResource: "SDCAlertView", withExtension: "bundle")
return bundleURL.map(Bundle.init(url:)) ?? frameworkBundle
```

Comment (maks-g) [reply Source/Bundle+Convenience.swift]: Return type for `bundleURL.map(Bundle.init(url:)) ?? sourceBundle` is 'Bundle??', so it requires additional unwrapping.

Comment (sberrevoets) [reply Source/Bundle+Convenience.swift]: Ah oops, that should be a `flatMap` then

**Comment by maks-g** [comment]
Updated, pls check.

**Pull request merged**